src/heap-inl.h

```diff
@@ -217,7 +217,7 @@ HEAP_EXPORT(void heap_remove(struct heap* heap,
    * It's a min heap so parent < child must be true.  If the parent is bigger,
    * swap it with the smallest child.
    */
-  for (;;) {
+  while (1) {
```

Comment:
Can you undo this stylistic change? `for (;;)` is the dominant pattern for infinite loops in libuv.

Reply:
Ohh, didn’t know about this. 👍 Done